cmd/opampsupervisor/README.md

```diff
@@ -1,5 +1,8 @@
 # OpAMP Supervisor for the OpenTelemetry Collector
 
+<!-- status autogenerated section -->
```

Comment:
Can you run `make -C cmd/opampsupervisor generate` to fill out this section?

Reply:
thanks done : )